types/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "lib": ["es2015", "dom"],
```

Comment:
Do we use the DOM btw? Never wrote a tsconfig so not sure if it’s needed? 🤷‍♂️

Reply:
We don't need it. And it doesn't matter for users because `tsconfig.json` is used for the type testing only. I'll delete it too.